src/core/Table/filters/customFilterFunctions.ts

```diff
@@ -0,0 +1,38 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { IdType, Row } from 'react-table';
+
+const isValidDate = (date: Date | undefined) =>
+  !!date && !isNaN(date.valueOf());
+
+const betweenDate = <T extends Record<string, unknown>>(
```

Comment:
Name sounds weird... Does it return values between min and max?

Reply:
Default one from react-table is `between` which is for numbers, so I went with the same naming. And yes it return values between min and max dates.